src/TabbedArea.js

```diff
@@ -105,12 +105,13 @@ const TabbedArea = React.createClass({
   },
 
   renderTab(child) {
-    let {eventKey, className, tab } = child.props;
+    let {eventKey, className, tab, disabled } = child.props;
```

Comment:
This child should be a `TabPane` if I'm not mistaken, can you add `disabled` to the list of `propTypes` on `TabPane`? This will ensure folks get warned if they use the wrong data type.